src/data/move.ts

```diff
@@ -8385,17 +8386,20 @@ export function initMoves() {
     new SelfStatusMove(Moves.RAGE_POWDER, Type.BUG, -1, 20, -1, 2, 5)
       .powderMove()
       .attr(AddBattlerTagAttr, BattlerTagType.CENTER_OF_ATTENTION, true),
-    new StatusMove(Moves.TELEKINESIS, Type.PSYCHIC, -1, 15, -1, 0, 5)
+    new StatusMove(Moves.TELEKINESIS, Type.PSYCHIC, 100, 15, -1, 0, 5)
```

Comment:
```suggestion
    new StatusMove(Moves.TELEKINESIS, Type.PSYCHIC, -1, 15, -1, 0, 5)
```
According to [Bulbapedia](https://bulbapedia.bulbagarden.net/wiki/Telekinesis_(move)), Telekinesis should bypass accuracy checks.